zebra/zebra_vty.c

```diff
@@ -3511,7 +3511,26 @@ DEFUN (show_evpn_mac_vni,
 
 	vni = strtoul(argv[4]->arg, NULL, 10);
 	zvrf = zebra_vrf_get_evpn();
-	zebra_vxlan_print_macs_vni(vty, zvrf, vni, uj);
+	zebra_vxlan_print_macs_vni(vty, zvrf, vni, uj, false);
+	return CMD_SUCCESS;
+}
+
+DEFPY (show_evpn_mac_vni_detail,
+       show_evpn_mac_vni_detail_cmd,
+       "show evpn mac vni " CMD_VNI_RANGE " detail [json]",
+       SHOW_STR
+       "EVPN\n"
+       "MAC addresses\n"
+       "VxLAN Network Identifier\n"
+       "VNIs number\n"
```

Comment:
VNI_NUM_HELP_STR

Reply:
Fixed

Reply:
Couldn't use the VNI_NUM_HELP_STR since it's defined in bgp_evpn_vty.h. Including this .h in zebra_vty.c results in compilation failure. So had to revert the fix.